skrifa/src/metrics.rs

```diff
@@ -213,6 +218,7 @@ impl Metrics {
 /// Glyph specific metrics.
 #[derive(Clone)]
 pub struct GlyphMetrics<'a> {
+    font: &'a FontRef<'a>,
```

Comment:
FontRef is small enough that I’d prefer to just clone it and store by value here to avoid the extra lifetime.